tfhe/src/integer/server_key/radix_parallel/div_mod.rs

```diff
@@ -113,134 +114,241 @@ impl ServerKey {
         //     Q(i) := 1
         //   end
         // end
+        assert_eq!(
```

Comment:
why all the asserts compared to before ?

Reply:
To be more explicit on the requirements of the imputs, before some of them were implicit or assert later (like in the comparisons)